client/src/app/tabs/form/__tests__/FormEditorSpec.js

```diff
@@ -815,6 +815,29 @@ describe('<FormEditor>', function() {
         expect(calls).to.have.lengthOf(1);
       });
 
+
+      it('should NOT break application on update', async function() {
+
+        // given
+        const props = {
+          layout: {
+            panel: {
+              open: true,
+              tab: 'linting'
+            }
+          }
+        };
+
+        // when
+        try {
+          await renderEditor(schema, props);
+        } catch (error) {
+
+          // then
+          expect(error).not.to.exist;
+        }
```

Comment:
This test still succeeds when the function throws, but no error is created (e.g. `throw null;`)
Unfortunately, `expect().not.to.throw` does not work with async functions. We could ensure it fails on all throws like this

```suggestion
        try {
          await renderEditor(schema, props);
        } catch (error) {

          // then
          // should not throw
          expect(true).to.false;
        }
```

Reply:
Bit of an edge case, as we would expect an error to be always thrown

Reply:
What we want to do is this:

```
let err;

try { something } catch (error) { err = error }

// then
expect(err).to.something;
```

Reply:
I'll rewrite it though I have not encountered `throw null;` yet.

Reply:
> I'll rewrite it though I have not encountered throw null; yet.

Throwing `null` or `undefined` would be worst practice 😄